examples/Prioritized_DQN/atari/alg.py

```diff
@@ -0,0 +1,145 @@
+#   Copyright (c) 2020 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import copy
+
+import numpy as np
+import paddle.fluid as fluid
+
+import parl
+from parl.core.fluid import layers
+from parl.core.fluid.algorithm import Algorithm
+
+
+class PrioritizedDQN(parl.Algorithm):
```

Comment:
Please put the file to the algorithm folder so that users can import it from parl.algorithm.
folder: parl/algorithms/fluid